react/react1/week1/react-week1/src/App.js

```diff
@@ -0,0 +1,20 @@
+const Todos = [
+  {id: 1, description: "Get out of bed", deadline: "Wed Sep 13 2017" }, 
+  {id: 2, description: "Brush teeth", deadline: "Thu Sep 14 2017"}, 
+  {id: 3, description: "Eat breakfast", deadline: "Fri Sep 15 2017" }
+  // {id: 4, description: "Eat launch", deadline: "Fri Sep 16 2017" }
+];
+
+
+export default function Todo(){
+  return (
+    <div className='Todo'> { 
+      Todos.map((list) => (
+        <div>
+  <h4> {`Id: ${list.id}`}</h4>
+  <h4> {`Description: ${list.description}`}</h4>
+  <h4> {`Deadline: ${list.deadline}`}</h4>
+</div>
+  ))}
+      </div> 
+  )};
```

Comment:
try to make separate file for array and logic of rendering list list from array.  fx. I think good idea to create separate component and also as per Assignment it is require to create at least two components in task.